src/app/components/RemainingTime.js

```diff
@@ -0,0 +1,34 @@
+function handleTimeChange(element){
+  let timer = 120;
+
+  const idInterval = setInterval(() => {
+    timer--;
+
+    if(timer === 0){
+      clearInterval(idInterval);
+    }
+
+    let min = Math.floor(timer / 60);
+    let sec = timer % 60;
+
+    element.textContent = `Time Left: ${min}m ${sec}s`;
+  }, 1000);
+};
+
+function RemainingTime() {
+  const root = document.querySelector("#swquiz-app");
+
+  const parentElement = document.createElement("div");
+  parentElement.classList.add("remaining-time");
+
+  const counter = document.createElement("div");
+  counter.textContent = "Time Left : 2 m 00 s";
+  counter.classList.add("remaining-time__counter");
+
```

Comment:
It's a mess here. First, declaration of all variables, then methods (classList.add, textContent).

Reply:
Maybe the timer will look better if 6s show as 06 sec (+ zero in front)?

Reply:
(for minutes too)

Reply:
> It's a mess here. First, declaration of all variables, then methods (classList.add, textContent).

In my opinion my way of formating is more transparent :)